tests/linux_kernel/__init__.py

```diff
@@ -213,7 +213,7 @@ def smp_enabled():
     return bool(re.search(r"\bSMP\b", os.uname().version))
 
 
-def parse_range_list(s):
+def parse_range_list(s) -> Set[int]:
```

Comment:
You can drop this since it won't be needed anymore once you use `parse_cpuspec()` instead.